docs/advanced-config.md

```diff
@@ -13,15 +13,16 @@ Below you will find all the configuration options supported by this distribution
+| `splunk.access.token`                   | `SPLUNK_ACCESS_TOKEN`                   | unset                   | Stable       | (Optional) Auth token allowing exporters to communicate directly with the Splunk cloud, passed as `X-SF-TOKEN` header. Currently, the [Jaeger trace exporter](#trace-exporters) as well as the [SignalFx metrics exporter](metrics.md) support this property. |
+| `splunk.realm`                          | `SPLUNK_REALM`                          | `none`                  | Stable       | The Splunk cloud realm where the telemetry should be sent to; e.g. `us0`, `us1`. Defaults to `none`, which means sending data to a Splunk OpenTelemetry Collector deployed on `localhost`.                                                                    |
+| `splunk.metrics.enabled`                | `SPLUNK_METRICS_ENABLED`                | `false`                 | Experimental | Enables exporting metrics. See [this document](metrics.md) for more information.                                                                                                                                                                              |
+| `splunk.metrics.endpoint`               | `SPLUNK_METRICS_ENDPOINT`               | `http://localhost:9943` | Experimental | The SignalFx metrics endpoint to connect to.  Setting this will override the `splunk.realm` property.                                                                                                                                                         |
+| `splunk.metrics.export.interval`        | `SPLUNK_METRICS_EXPORT_INTERVAL`        | `30s`                   | Experimental | The interval between pushing metrics.  <details><summary>Format</summary>Durations can be of the form `{number}{unit}`, where unit is one of `ms`, `s`, `m`, `h`, `d`. If no unit is specified, milliseconds is the assumed duration unit.</details>          |
+| `splunk.metrics.implementation`         | `SPLUNK_METRICS_IMPLEMENTATION`         | `micrometer`            | Experimental | The metrics implementation used by the agent. Valid values are `micrometer` and `opentelemetry`.                                                                                                                                                              |
+| `splunk.metrics.force_full_commandline` | `SPLUNK_METRICS_FORCE_FULL_COMMANDLINE` | `false`                 | Experimental | Forces the full commandline to be included as a resource attribute for all metrics. Default truncates to 255 chars.                                                                                                                                           |
```

Comment:
```suggestion
| `splunk.metrics.force_full_commandline` | `SPLUNK_METRICS_FORCE_FULL_COMMAND_LINE` | `false`                 | Experimental | Adds the full command line as a resource attribute for all metrics. Maximum length is 255 characters. Commands longer than 255 characters are truncated.                                                                                                                                           |
```

@breedx-splk It's `COMMAND_LINE` according to the code, not `COMMANDLINE`, right? By the way, what do we refer to with that? The full JVM invocation?

Reply:
I admit, it's confusing. The code constant has `COMMAND_LINE` but the system property has `COMMANDLINE` because commandline is one word for now and forever in my brain and you won't convince me otherwise. 🙃  This markdown is intended to document to the user what the actual config item is, not the code constant.

Reply:
Haha, that's perfectly OK. Just wanted to be sure. :)